bpf/bpf_host.c

```diff
@@ -844,6 +844,15 @@ do_netdev(struct __ctx_buff *ctx, __u16 proto, const bool from_host)
 	__u32 __maybe_unused ipcache_srcid = 0;
 	int ret;
 
+	if (from_host) {
```

Comment:
Just a thought: there's quite a few `if (from_host)` branches here. Might make sense to collapse them and perhaps move the bodies into inlined functions?

Reply:
Also it might sense to put this feature behind an `#ifdef` if possible as it does seem to add quite a few branches to the load-balancer hot path.

Reply:
Adding `#ifdef`, but will leave more far reaching refactoring for later.

Reply:
Thanks!